app/views/shared/_filter.html.erb

```diff
@@ -1,16 +1,36 @@
-<%= search_form_for @q do |f| %>
 
-<%= f.label :postcode %>
-<%= f.search_field :address_cont %>
+<%= search_form_for @q, html: {class:"row justify-content-center align-items-center pb-3"} do |f| %>
 
-<%= f.label :max_price %>
-<%= f.search_field :price_lteq %>
+<div class="form-group col-auto">
+<%= f.label :postcode, class:"col-form-label" %>
+</div>
+<div class="col-auto">
+<%= f.search_field :address_cont, class:"form-control"%>
+</div>
 
-<%= f.label :min_price %>
-<%= f.search_field :price_gteq %>
+<div class="form-group  col-auto">
+<%= f.label :min_price, class:"col-form-label" %>
+</div>
+<div class="col-auto">
+<%= f.search_field :price_gteq, class:"form-control" %>
+</div>
 
-<%= f.label :collection_date %>
-<%= f.date_field :bookings_collection_date_not_eq %>
+<div class="form-group col-auto">
+<%= f.label :max_price, class:"col-form-label" %>
+</div>
+<div class="col-auto">
+<%= f.search_field :price_lteq, class:"form-control" %>
+</div>
+
+<div class="form-group col-auto">
+<%= f.label :collection_date, class:"col-form-label" %>
+</div>
+<div class="col-auto">
```

Comment:
add pt-3 here to align with the submit button after it.

Reply:
👍🏻